### DISCUSSION TITLE
Circular view "crash"

### DISCUSSION THREAD

**Discussion by jrobinso** [GMOD/jbrowse-components#2464] [General] [answered]
Hi @garrettjstevens ,  I'm not sure if this is really a bug or a consequence of how we're using the circular view so I won't add to your 258 issues unless you want.   If I click the "hamburger" menu icon in the circ view I get a tracks dialog.  So far so good, but if I click "About Track" (see menu) the circ view screen goes white and there's no way to recover, it has to be closed.  I don't really need this functionality but if its there someone will click it.    See the screenshot of the stack trace below.   The question: is this a consequence of how we are initializing it?  I'm following the instructions you sent, if you need something to reproduce it I can provide instructions.

<img width="606" alt="Screen Shot 2021-10-27 at 1 14 39 PM" src="https://user-images.githubusercontent.com/933148/139140263-dc683cc9-efba-442d-a528-38e66482711f.png">

<img width="1098" alt="Screen Shot 2021-10-27 at 12 30 54 PM" src="https://user-images.githubusercontent.com/933148/139140693-c9f730a3-5e05-4674-ac8e-f0a741327f7b.png">

**Comment by garrettjstevens** [comment, marked as answer]
Hi @jrobinso. This is indeed a bug, and I can reproduce it. I'll go ahead and test out the "Create issue from discussion" button here (I haven't tried it before) and create an issue for us to track. Thanks.

**Reply by rbuels** [reply]
issue is #2466

**Reply by rbuels** [reply]
PR with the fix is #2469
